### DISCUSSION TITLE
Any plans on fixing that it adds your login automatically when you open a place with a search box?

### DISCUSSION THREAD

**Discussion by biast12** [victornpb/undiscord#303] [1. Questions & Answers]
Any plans on fixing that it adds your login automatically when you open a place with a search box? it's getting really annoying having to remove that every time i wanna post an emoji

**Comment by Toby222** [comment]
Could you elaborate?

What do you mean?
What browser are you using?

**Reply by biast12** [reply]
Every time i open a place where it got a search box does it automatically add my login username, as you can see in this video when i open the emoji menu: 
https://user-images.githubusercontent.com/53872542/149152449-dbdfaa80-3d58-4b88-a6a9-c86b57c23871.mp4

it's honestly making the script / tool unusable for me

**Reply by biast12** [reply]
ohh yeah btw this is using Violentmonkey in opera

**Reply by Toby222** [reply]
Apparently this is a problem with "modern browsers", who decided that they should ignore the `autocomplete` attribute on input elements for their internal password managers.
There's no way to tell the browser not to fill the token field, or that the search fields are *not*, in fact, a username field for the token field.

I think this is more of a bug in browsers than in the userscript.
For developer, [there doesn't seem to be a single accepted solution..](https://stackoverflow.com/questions/40200130/how-to-disable-chrome-to-autofill-username-email-password/40200269#40200269)

**Reply by victornpb** [reply]
That is REALLY odd, it seems that their heuristics is completely broken. If i understand it right, the browser thinks the gif box is a login form because there's a password field on a completely different form which is hidden?! 🥴

I know the Safari team at apple has a form for patching the password manager heuristics for specific websites, maybe opera has something like this too... anyways theres not much we can do on our side apart from trying to fool Opera.

On a major rewrite of undiscord I want to make most of options fully automated, but for now the password field is there for the general public security. Most people doesn't understand [how sensible the token is](https://github.com/victornpb/deleteDiscordMessages#do-not-share-your-authtoken), and it should be treated as a password.

**Reply by biast12** [reply]
Yep! @victornpb that's right, it ses the gif/stickers/emojis search bar as a login form when i'm using the Undiscord addon (via Violentmonkey), and yes it's only when having Undiscord activated

**Reply by biast12** [reply]
ohh i just checked and it looks like it's working normally now, i'll report back if it starts doing it again, tho for now really thanks for this! it's maybe things so much easier

**Reply by victornpb** [reply]
Ok, thank you!
